app/models/task.rb

```diff
@@ -31,6 +31,7 @@ class Task < CaseflowRecord
 
   before_create :set_assigned_at
   before_create :verify_org_task_unique
+  before_create :set_dispatch_blocker_parent, if: :blocking_dispatch?
```

Comment:
This method name makes more sense to me:
```suggestion
  before_create :set_dispatch_as_parent, if: :blocking_dispatch?
```

Reply:
Combined both your suggestions to `conditionally_set_dispatch_as_parent`